src/hip_memory.cpp

```diff
@@ -955,37 +955,14 @@ hipError_t hipHostUnregister(void* hostPtr) {
     return ihipLogStatus(hip_status);
 }
 
-namespace {
-inline hipDeviceptr_t agent_address_for_symbol(const char* symbolName) {
```

Comment:
@AlexVlx `agent_address_for_symbol()` is now removed, and we use `hipGetSymbolAddress()` instead, right?